app/views/tasks/_nested_file_form.html.slim

```diff
@@ -2,21 +2,21 @@
   = f.nested_fields_for :files do |file|
     .show-table.exercise-show
       - if file.object.new_record?
-        - display_container = 'display: block' # rubocop:disable Lint/UselessAssignment
+        - display_container = 'display: block'
         .table-header
           = file.remove_nested_fields_link :text, class: 'btn btn-main btn-sm float-end' do
             i.fa-solid.fa-xmark.text-danger
             =< t('.remove_file')
       - else
-        - display_container = 'display: none' # rubocop:disable Lint/UselessAssignment
+        - display_container = 'display: none'
         .table-header.closed.toggle-next
           => file.object.full_file_name
           span.fa-solid.fa-caret-down
           = file.remove_nested_fields_link :text, class: 'btn btn-main btn-sm float-end' do
             i.fa-solid.fa-xmark.text-danger
             =< t('.remove_file')
-      .file-container.mx-2.my-3 style=display_container
-        = render 'file_form', f:, file:, nested_form_placeholder:
+      .file-container.mx-2.my-3 style="#{display_container}"
```

Comment:
This change is currently being discussed as part of #1527, I think. Let's see how we decide there. For now, I would probably not touch it (neither here nor the few occurrences above) if possible.